runtime/src/test_utils.rs

```diff
@@ -576,6 +580,9 @@ impl<BS: Blockstore> MockRuntime<BS> {
             value,
             send_return,
             exit_code,
+            send_flags: SendFlags::default(),
+            gas_limit: None,
+            send_error: None,
```

Comment:
Nit: move flags and gas limit, which are inputs, above send_return etc outputs. I would also move the send_error above exit_code, as it conceptually happens first.